spec/models/ems_refresh/save_inventory/save_tags_inventory_spec.rb

```diff
@@ -6,7 +6,7 @@ def mapped_tag(category_name, tag_name)
     mapping = FactoryGirl.create(:tag_mapping_with_category,
                                  :category_name        => category_name,
                                  :category_description => category_name)
-    category = mapping.tag.category
+    category = mapping.tag.classification
```

Comment:
Why does this have to change?  The code is looking for the category, so it's surprising to ask for the classification

Reply:
`category` currently returns a `category` or `classification`

Spoke with Beni about this and he agreed it was relying upon a bug in `#category`. Changing this line still passes tests. 

In this test, there is category/tag (which really translates to classification/category)